guides/common/modules/proc_creating-a-remote-source-for-a-custom-file-type-repository.adoc

```diff
@@ -21,6 +21,14 @@ For more information about configuring a web server, see {RHELDocsBaseURL}9/html
 endif::[]
 
 .Procedure
+ifdef::katello[]
+. On your server, enable the required repositories:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# dnf config-manager --add-repo=https://yum.theforeman.org/pulpcore/{PulpcoreVersion}/el9/x86_64/
```

Comment:
This does not work as is:

````
Public key for python3.11-pulp_manifest-3.0.0-5.el9.noarch.rpm is not installed
The downloaded packages were saved in cache until the next successful transaction.
You can remove cached packages by executing 'dnf clean packages'.
Error: GPG check FAILED
````

My reproducer:

````
$ podman run --rm -it almalinux:9.5

# dnf install -y 'dnf-command(config-manager)'
# dnf config-manager --add-repo=https://yum.theforeman.org/pulpcore/3.63/el9/x86_64/
# echo "gpgkey=https://yum.theforeman.org/pulpcore/3.63/GPG-RPM-KEY-pulpcore" >> /etc/yum.repos.d/yum.theforeman.org_pulpcore_3.63_el9_x86_64_.repo 
# dnf makecache && dnf install -y pulp-manifest
# pulp-manifest --help
````

So either we add the `echo "..." >> pulp.repo` command or use `echo` to write the whole repo file. I do not think that `dnf config-manager` to setup the URL for the GPG pub key.

Reply:
I vote we add
```
# echo "gpgkey=https://yum.theforeman.org/pulpcore/3.63/GPG-RPM-KEY-pulpcore" \
>> /etc/yum.repos.d/yum.theforeman.org_pulpcore_3.63_el9_x86_64_.repo
```

That was the only extra line I needed to get it working.

Reply:
(replacing the 3.63 with `{PulpcoreVersion}`)

Reply:
> {PulpcoreVersion}

Updated.